### DISCUSSION TITLE
I'm using brew for some time. I want to ask how it is installed.

### DISCUSSION THREAD

**Discussion by darkRaspberry** [Homebrew/discussions#2339] [Everyday usage]
When installing, brew download few repo and then try to resolve all the deltas. Doesn't we have some more less complicated method. Resolving those deltas take time and space.

If the core members can suggest somethings about it.

**Comment by SMillerDev** [comment]
This is the simplest method because it's all left to git to do versioning.

**Reply by darkRaspberry** [reply]
Well sounds promising. Let me re-understand the installation process.
On surface level doesn't only the source file is required for setting up the brew.

**Reply by darkRaspberry** [reply]
*Latest* source file.
